.changeset/spotty-bottles-rest.md

```diff
@@ -0,0 +1,109 @@
+---
+"fluid-framework": minor
+"@fluidframework/tree": minor
+"__section": feature
+---
+Persisted metadata for Shared Tree schemas
+
+The persisted metadata feature for Shared Tree allows an application author to write metadata along with the schema. This feature is supported for both object node and field schemas.
```

Comment:
```suggestion
The persisted metadata feature for Shared Tree allows an application author to write document-persisted metadata along with the schema. This feature is supported for both object node and field schemas.
```

...or something to that effect. The concept of having metadata on your schema was already a thing, as you know :)